template_content/smart_contracts/config.py.jinja

```diff
@@ -14,14 +13,15 @@ from algokit_utils import (
 from algosdk.util import algos_to_microalgos
 from algosdk.v2client.algod import AlgodClient
 from algosdk.v2client.indexer import IndexerClient
-{% endif %}
+
 from smart_contracts import helloworld
+from smart_contracts.artifacts.HelloWorldApp import client
```

Comment:
```suggestion
from smart_contracts.artifacts.HelloWorldApp.client import HelloWorldAppClient
```

This line should also be wrapped in a `{% if deployment_language == 'python' %}`